dashboards/lodestar_rest_api.json

```diff
@@ -657,7 +679,7 @@
   },
   "timezone": "utc",
   "title": "Lodestar - REST API",
-  "uid": "Lodestar_rest_api",
+  "uid": "lodestar_rest_api",
```

Comment:
Updated uid to ensure it is picked up by download script
https://github.com/ChainSafe/lodestar/blob/2493672e9e26d202e12c657a35b8a5bd7fae972a/scripts/download_dashboards.mjs#L33